### PR TITLE
docs: add new team members

### PR DESCRIPTION
**Pull request by userquin**
This PR includes 2 new team members: 
- fix missing type from pwa module
- simplify core team type
- Vitest webtoo.ls on social links (nav bar)
- Mastodon links for some folks
- Discord and YouTube social links for Anthony

/cc @poyoho @AriPerkkio can you review your data?
- Ari: can you tell me a small description and if twitter account is ok?
- poyoho: check your name and the description

~~I'll check if we can also include another social entries, IIRC there is something on VitePress to do this...~~

### CODE REVIEW

File: docs/.vitepress/contributors.ts [hunk 1]
```diff
@@ -103,6 +103,22 @@ const plainTeamMembers: CoreTeam[] = [
...
+    title: 'It\'s no problem in my locall',
+    desc: 'Core team member of Vite & Vitest',
+  },
+  {
+    avatar: contributorsAvatars.AriPerkkio,
+    name: 'Ari Perkkiö',
+    github: 'AriPerkkio',
+    twitter: '@perkkio_ari',
+    title: 'Core team member of Vitest, working',
+    org: 'Cloudamite',
```
Comment (AriPerkkio) [inline docs/.vitepress/contributors.ts:120]:
```suggestion
    title: 'A fullstack developer, working',
    org: 'Cloudamite',
    desc: 'Core team member of Vitest',
```

File: docs/.vitepress/contributors.ts [hunk 2]
```diff
@@ -103,6 +103,22 @@ const plainTeamMembers: CoreTeam[] = [
...
+    github: 'poyoho',
+    twitter: '@yoho_po',
+    title: 'It\'s no problem in my locall',
+    desc: 'Core team member of Vite & Vitest',
+  },
+  {
+    avatar: contributorsAvatars.AriPerkkio,
+    name: 'Ari Perkkiö',
+    github: 'AriPerkkio',
+    twitter: '@perkkio_ari',
```
Comment (AriPerkkio) [inline docs/.vitepress/contributors.ts:118]: I don't actually have twitter or any other similar platforms at all. Just Github and email. 🙃 

The typings and logic seem to require some refactoring to make this field optional.
```suggestion
```

**Review by AriPerkkio — COMMENTED** [review]
Cool, thanks! By the way when trying this locally, I got HTTP429 rate limit hit from Github. It seems the avatars are requested on every HMR.

**Comment by userquin** [comment]
> Cool, thanks! By the way when trying this locally, I got HTTP429 rate limit hit from Github. It seems the avatars are requested on every HMR.

It is no yet official, I missunderstood a message...

Unzip public.zip inside `docs/public` folder, then run `nr docs` or `pnpm docs`: you can also test it using `nr docs:https-no-prefetch` or `pnpm docs:https-no-prefetch`. Open `https://localhost` in private window, the PWA will be removed once you close the private browsing window.

[public.zip](https://github.com/vitest-dev/vitest/files/10459574/public.zip)

File: docs/.vitepress/contributors.ts [hunk 3]
```diff
@@ -83,6 +99,7 @@ const plainTeamMembers: CoreTeam[] = [
     avatar: contributorsAvatars.Demivan,
     name: 'Ivan Demchuk',
     github: 'Demivan',
+    fosstodon: 'demivan',
     twitter: 'IvanDemchuk',
```
Comment (Demivan) [inline docs/.vitepress/contributors.ts:103]: You can drop it, while you are at it. I have deleted my twitter account.
```suggestion
```

**Comment by userquin** [comment]
@antfu @patak-dev review NuxtLabs and StackBlitz links...

**Review by antfu — APPROVED** [review]
My part is good, thanks!

File: docs/.vitepress/contributors.ts [hunk 4]
```diff
@@ -32,12 +37,26 @@ export const contributors = (contributorNames as string[]).reduce((acc, name) =>
...
+    tm.links.push({ icon: 'mastodon', link: `https://elk.zone/hachyderm.io/@${tm.hachyderm}` })
+
+  if (tm.discord)
+    tm.links.push({ icon: 'discord', link: tm.discord })
+
+  if (tm.youtube)
+    tm.links.push({ icon: 'youtube', link: `https://www.youtube.com/@${tm.youtube}` })
+
+  if (tm.twitter)
+    tm.links.push({ icon: 'twitter', link: `https://twitter.com/${tm.twitter}` })
```
Comment (antfu) [inline docs/.vitepress/contributors.ts:58]: Can we use a `mastodon` prop with a full link as the value here?

Comment (userquin) [reply docs/.vitepress/contributors.ts:58]: sure, so like discord one...

Comment (userquin) [reply docs/.vitepress/contributors.ts]: done

**Comment by userquin** [comment]
@zxch3n can you check your mastodon link (avatar seems match with twitter)?

**Comment by userquin** [comment]
the preview can be found here: https://deploy-preview-2715--vitest-dev.netlify.app/

File: docs/.vitepress/contributors.ts [hunk 5]
```diff
@@ -99,10 +107,28 @@ const plainTeamMembers: CoreTeam[] = [
     avatar: contributorsAvatars.zxch3n,
     name: 'Zixuan Chen',
     github: 'zxch3n',
+    mastodon: 'https://elk.zone/hachyderm.io/@zx',
     twitter: 'zxch3n',
     title: 'A fullstack developer',
     desc: 'Creating tools for collaboration',
```
Comment (zxch3n) [inline docs/.vitepress/contributors.ts:113]: My mastodon link is correct. Can you help me update this to “Working on CRDTs & local-first software”

Comment (userquin) [reply docs/.vitepress/contributors.ts:114]: check latest changes

Comment (zxch3n) [reply docs/.vitepress/contributors.ts:114]: Thanks, but I meant:

Title: A fullstack developer
Desc: Working on CRDTs & local-first software

Comment (userquin) [reply docs/.vitepress/contributors.ts]: done

**Review by zxch3n — APPROVED** [review]

**Review by poyoho — APPROVED** [review]

**Pull request merged**